src/olympia/amo/management/commands/fix_licenses.py

```diff
@@ -0,0 +1,68 @@
+from django.core.management.base import BaseCommand
+from django.utils.translation import activate
+
+from olympia.versions.models import License
+from olympia.addons.models import Addon
+from olympia.devhub.models import ActivityLog
+from olympia.translations.models import Translation
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        activate('en-us')
+
+        mpl11 = License.objects.get(pk=5)
+
+        assert str(mpl11.name) == 'Copyright Jason Savard', 'wrong license'
+
+        # set on_form explicitly to False to make sure it's not
+        # visible and set `builtin` to 1 to make sure it's filtered
+        # everywhere.
+        mpl11.on_form = False
+        mpl11.builtin = 1
+        mpl11.save()
+
+        search = '"versions.license": 5}'
+        changed_addons = ActivityLog.objects.filter(
+            action=37,
+            _arguments__contains=search)
+
+        msg = 'too many or few add-ons changed, found %s' % len(changed_addons)
+        assert len(changed_addons) == 3, msg
```

Comment:
No, there were 3 add-on entries we found that actually changed their licnese text after the degredation of mpl1.1 happened.